Gopkg.toml

```diff
@@ -101,7 +106,3 @@ required = [
   [[prune.project]]
     name = "github.com/ligato/cn-infra"
     unused-packages = false
-
-  [[prune.project]]
```

Comment:
This served as a workaround for `dep check` failure, which actually occurred in the last build on this PR:

```sh
dep check
# vendor is out of sync:
github.com/go-errors/errors: hash of vendored tree not equal to digest in Gopkg.lock
```